app/pandas.ipynb

```diff
@@ -333,114 +624,302 @@
-   "outputs": [],
    "source": [
-    "# use ** operator here"
+    "# use ** operator here\n",
+    "s1 ** s2"
    ],
    "metadata": {
-    "collapsed": false
-   }
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-10-30T13:31:50.642151Z",
+     "start_time": "2024-10-30T13:31:50.634141Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0          1\n",
+       "1        128\n",
+       "2       6561\n",
+       "3     262144\n",
+       "4    9765625\n",
+       "dtype: int64"
+      ]
+     },
+     "execution_count": 31,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 31
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# use // operator here"
+    "# use // operator here\n",
+    "s1 // s2"
    ],
    "metadata": {
-    "collapsed": false
-   }
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-10-30T13:32:11.394978Z",
+     "start_time": "2024-10-30T13:32:11.388029Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0    0\n",
+       "1    0\n",
+       "2    0\n",
+       "3    0\n",
+       "4    0\n",
+       "dtype: int64"
+      ]
+     },
+     "execution_count": 32,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 32
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# use % operator here"
+    "# use % operator here\n",
+    "s1 % s2"
    ],
    "metadata": {
-    "collapsed": false
-   }
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-10-30T13:32:18.611757Z",
+     "start_time": "2024-10-30T13:32:18.603940Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0    1\n",
+       "1    2\n",
+       "2    3\n",
+       "3    4\n",
+       "4    5\n",
+       "dtype: int64"
+      ]
+     },
+     "execution_count": 33,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 33
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# filter s1 here"
+    "# filter s1 here\n",
+    "s1[s1 % 2 == 0]"
    ],
    "metadata": {
-    "collapsed": false
-   }
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-10-30T13:32:47.885785Z",
+     "start_time": "2024-10-30T13:32:47.878722Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "1    2\n",
+       "3    4\n",
+       "dtype: int64"
+      ]
+     },
+     "execution_count": 34,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 34
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# filter s2 here"
+    "# filter s2 here\n",
+    "s2[s2 % 2 == 0]"
```

Comment:
The filter for s2 is supposed to print only odd numbers, but the condition `s2 % 2 == 0` filters for even numbers. Change the condition to `s2 % 2 != 0` to filter odd numbers.